rust/src/lib.rs

```diff
@@ -28,3 +28,10 @@ pub unsafe extern "C" fn evmrs_dump_coverage() {
 pub extern "C" fn evmrs_is_coverage_enabled() -> u8 {
     cfg!(feature = "dump-cov") as u8
 }
+
+#[cfg(feature = "mimalloc")]
+use mimalloc::MiMalloc;
+
+#[cfg(feature = "mimalloc")]
+#[global_allocator]
+static GLOBAL: MiMalloc = MiMalloc;
```

Comment:
is this it? is this everything you need to change the system allocator?

How does it work?  is global a "keyword" symbol?

Reply:
`#[global_allocator]` is a compiler build-in which registers a static as a global allocator. The name of the static does not matter.